test/ably/restchannelpublish_test.py

```diff
@@ -508,3 +508,28 @@ def side_effect(self, *args, **kwargs):
 
         assert failures == 2
         assert len(channel.history().items) == 1
+
+    # RSL1k5
```

Comment:
I don't think this test is correct.  We are not talking about error conditions in the spec, we are requiring that the same message is published many times successfully, and then we confirm only one message is stored.